File: python/2310.py
quantidade_jogadores = int(input())
total_s, total_b, total_a = 0, 0, 0
s1, b1, a1 = 0, 0, 0

for i in range(quantidade_jogadores):
    nome = input()
    s, b, a = map(int, input().split())
    acertos_s, acertos_b, acertos_a = map(int, input().split())
    total_s += s
    total_b += b
    total_a += a
    s1 += acertos_s
    b1 += acertos_b
    a1 += acertos_a

print("Pontos de Saque: %.2f %%." %(s1/total_s * 100))
print("Pontos de Bloqueio: %.2f %%." %(b1/total_b * 100))
print("Pontos de Ataque: %.2f %%." %(a1/total_a * 100))

File: dart/2310.dart
import 'dart:io';

void main() {
  int n = int.parse(stdin.readLineSync()!);
  int totalS = 0, totalB = 0, totalA = 0;
  int s = 0, b = 0, a = 0;

  for (int i = 0; i < n; i++) {
    stdin.readLineSync()!;
    List<String> t = stdin.readLineSync()!.split(" ");
    List<String> x = stdin.readLineSync()!.split(" ");

    totalS += int.parse(t[0]);
    totalB += int.parse(t[1]);
    totalA += int.parse(t[2]);

    s += int.parse(x[0]);
    b += int.parse(x[1]);
    a += int.parse(x[2]);
  }

  print(
      "Pontos de Saque: ${(s / totalS * 100).toStringAsFixed(2)} %.\nPontos de Bloqueio: ${(b / totalB * 100).toStringAsFixed(2)} %.\nPontos de Ataque: ${(a / totalA * 100).toStringAsFixed(2)} %.");
}
